assets/component-localization-form.css

```diff
@@ -234,11 +236,10 @@ noscript .localization-selector.link {
 
 .header-localization:not(.menu-drawer__localization) {
   align-items: center;
-  min-height: 4.4rem;
 }
 
 .js .header-localization:not(.menu-drawer__localization) .localization-form__select {
-  padding: 0 2.7rem 0 1.2rem;
+  padding: 0.85rem 2.7rem 0.85rem 1.2rem;
```

Comment:
Not a big fan of the kinda magic numbers. I'd rather set the desired height of 38px more transparently and align things with flex box, but if that would require additional refactoring to accommodate than this works for now.

Reply:
Deeeeefinitely. I don't know why I bothered trying to figure out these weird `rem` values. 😂 Fixed in dff48f2.